pkg/controller/master/virtualmachine/vmi_controller.go

```diff
@@ -17,11 +17,29 @@ import (
 
 const (
 	VirtualMachineCreatorNodeDriver = "docker-machine-driver-harvester"
+
+	AnnotationTopologyRegion = "topology." + builder.HarvesterAPIGroup + "/region"
```

Comment:
AnnotationTopologyKeyPrefix = "topology." + builder.HarvesterAPIGroup
AnnotationTopologyKeyRegion = AnnotationTopologyKeyPrefix  + "/region"
AnnotationtopologyKeyZone   = AnnotationTopologyKeyPrefix  + "/zone"
AnnotationTopologyKeyHost   = AnnotationTopologyKeyPrefix  + "/host"

Reply:
prefer to move it to the https://github.com/harvester/harvester/blob/master/pkg/util/constants.go